examples/Store/StoreSetLanguagesAction.json

```diff
@@ -0,0 +1,4 @@
+{
+  "action": "setLanguages",
+  "languages": ["languagesString"]
```

Comment:
Here I would suggest to use a correct value `["en"]` instead of the invalid `["languagesString"]`.

Reply:
yes, make sense ;) I'm updating it